diesel_cli/src/cli.rs

```diff
@@ -162,6 +166,16 @@ pub fn build_cli() -> App<'static, 'static> {
                 .possible_values(&Shell::variants()),
         );
 
+    let features_subcommand = SubCommand::with_name("features")
+        .about("list support for which database engines have been compiled into this version of diesel")
+        .arg(
+            Arg::with_name("features")
```

Comment:
I don't understand why you have an arg here. Do you want `features` as subcommand or arg?

Reply:
I was trying to get `diesel -f` to work but realised that is probably not desirable. I will remove this part.